### ISSUE TITLE
Failed to synchronize registry data from server, closing connection. 

### ISSUE THREAD

**Issue by camran131** [AllTheMods/ATM-6#1480]
HAMACHI SERVER ~

My buddy updated the server to the latest version 1.6.1 on curseforge. He copy/pasted the save to the new version from the old (1.5.7). Tried to connect and got the message "Failed to synchronize registry data from server, closing connection."
Any way to solve this?

**Comment by Magamelin** [comment]
Try to full reinstal minecraft, it helped me

**Comment by whatthedrunk** [comment]
Yeah seems it's best to always do a fresh install on update then copy the world over. If updating remotely install locally then delete the folders from the server and copy fresh over. Leaving stuff like the world folder alone.
